modules/account-lib/src/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,55 @@
+import { bigIntFromBufferBE, bigIntToBufferBE } from '../util';
+import BaseCurve from './baseCurve';
+import * as secp from '@noble/secp256k1';
+
+const order = secp.CURVE.n;
+
+export class Secp256k1Curve implements BaseCurve {
+  scalarRandom(): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.randomPrivateKey()));
+  }
+
+  scalarAdd(x: bigint, y: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateAdd(x, y.toString(16))));
```

Comment:
I just noticed that this needs to be `bigIntToBufferBE` not `.toString(16)`, as you rightly corrected on other functions.

Reply:
<img width="724" alt="image" src="https://user-images.githubusercontent.com/98419902/169394545-8a87842d-aedb-427f-9234-166e36d5fd54.png">
Seems like tweek is expecting a hex value ? @brandonblack